qiskit/extensions/standard/u1.py

```diff
@@ -232,3 +251,79 @@ def cu1(self, theta, control_qubit, target_qubit,
 
 
 QuantumCircuit.cu1 = cu1
+
+
+class MCU1Gate(ControlledGate):
+    r"""Multi-controlled-U1 gate.
+
+    This is a diagonal and symmetric gate that induces a
+    phase on the state of the target qubit, depending on the state of the control qubits.
+
+    **Circuit symbol:**
+
+    .. parsed-literal::
+
+            q_0: ────■────
+                     │
+                     .
+                     │
+        q_(n-1): ────■────
+                 ┌───┴───┐
+            q_n: ┤ U1(λ) ├
+                 └───────┘
```

Comment:
Is this how this actually gets rendered currently? It should be something like:
```
            q_0: ────■────
                     │ 
                     .
                     │         U1(λ)
        q_(n-1): ────■────
                     .
                     │
        q_(n):   ────■────

```

Because in this gate changing control and target will not make a difference. I know this works when there's only one control, but maybe not for more controls. If not, then can you open an issue? (It should b e fixed for all 3 types of drawer: latex, text, mpl)